styles.css

```diff
@@ -83,3 +104,149 @@ ul.socials {
   justify-content: space-between;
   max-width: 170px;
 }
+
+ul.works_grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 114px 24px;
+  grid-row-gap: 88px;
+  list-style-type: none;
+}
+
```

Comment:
- [ ] Great job with your design 💪. As seen in your chosen Figma design kindly give your work section a `background-color` 🙏

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206480223-60111b9c-aba7-4b30-b05a-9456158ab8de.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206480164-c55e98e7-bf1d-4fae-8c6e-9b50a9e235f7.png)

Reply:
Thanks for your feedback @Ugboaja-Uchechi. However, I've checked the Figma file and I can't seem to locate the background color for the work section.

Reply:
Hi @Ugboaja-Uchechi . About this, I just noticed it wasn't stated in the Figma file I duplicated in my drafts. However, it is on the original file. Thanks once again for highlighting this